Specs/DateTime/Swedish/TimeZoneExtractor.json

```diff
@@ -0,0 +1,360 @@
+[
+  {
+    "Input": "Boka ett rum åt mig Afghansk tid",
+    "NotSupportedByDesign": "javascript, python, java",
+    "Results": [
+      {
+        "Text": "Afghansk tid",
+        "Type": "timezone",
+        "Start": 20,
+        "Length": 12
+      }
+    ]
+  },
```

Comment:
But before we can mege the PR, we need to add the TimeZoneParser.json specs. To show that resolution also works.